sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/operation/GetColumnsOperation.java

```diff
@@ -171,7 +172,7 @@ public void runInternal() throws HiveSQLException {
           List<SQLPrimaryKey> primaryKeys = metastoreClient.getPrimaryKeys(new PrimaryKeysRequest(dbName, table.getTableName()));
           Set<String> pkColNames = new HashSet<>();
           for(SQLPrimaryKey key : primaryKeys) {
-            pkColNames.add(key.getColumn_name().toLowerCase());
+            pkColNames.add(key.getColumn_name().toLowerCase(Locale.ROOT));
```

Comment:
Same for column names. I don't think we should apply root locale

Reply:
Done